src/pages/version/ParametersModal/index.tsx

```diff
@@ -26,23 +35,34 @@ export const ParametersModal: React.FC<ParametersProps> = ({ parameters }) => {
         </MetadataItem>
       )}
       <DisplayModal
+        data-cy="parameters-modal"
         open={showModal}
         setOpen={setShowModal}
+        size="large"
         title="Patch Parameters"
-        data-cy="parameters-modal"
       >
-        {parameters?.map((param) => (
-          <StyledBadge key={`param_${param.key}`}>
-            {param.key}: {param.value}
-          </StyledBadge>
-        ))}
+        <OverflowContainer>
+          <BaseTable table={table} />
+        </OverflowContainer>
       </DisplayModal>
     </>
   );
 };
 
-const StyledBadge = styled(Badge)`
-  :not(:last-of-type) {
-    margin-right: ${size.s};
-  }
+const OverflowContainer = styled.div`
+  max-height: 600px;
+  overflow-y: scroll;
 `;
+
+const columns: LGColumnDef<Parameter>[] = [
+  {
+    accessorKey: "key",
+    header: "Key",
+    cell: (value) => value.getValue(),
```

Comment:
You can omit the cell field entirely because the accessor key exactly matches the field name and we don't need to render it in any special way
```suggestion
```